schema.sql

```diff
@@ -42,3 +42,32 @@ ADD COLUMN owner_id INT,
 ADD CONSTRAINT owner_key
   FOREIGN KEY (owner_id)
     REFERENCES owners(id);
+
+/* Create a table named vets */
+
+CREATE TABLE vets (
+  id serial PRIMARY KEY,
+  name TEXT,
+  age INT,
+  date_of_graduation DATE
+);
```

Comment:
- [ ] Great job so far, please try to correct your datatype and add the `PRIMARY KEY` as seen below:
`CREATE TABLE vets (`
  ` id int NOT NULL IDENTITY(1,1),`
   `name varchar(255),`
   `age int,`
  ` date_of_graduation date,`
   `PRIMARY KEY (id)
   );
`